package.json

```diff
@@ -28,6 +28,7 @@
     "lint": "concurrently yarn:lint:*",
     "lint:commit": "commitlint --from origin/main --to HEAD --verbose",
     "lint:es": "eslint . --ignore-path .gitignore --max-warnings 0",
+    "lint:ts": "tsc --noEmit",
```

Comment:
Should this also [run on CI](https://github.com/onfido/castor/blob/fdc2bcb6920b0fe49656fb40a1af40fc5f787797/.github/workflows/lint.yml#L6)?

Reply:
Definitely, I forget we don't do `yarn lint` to run all things. Will update.